assets/src/animation/parts/simpleAnimation.js

```diff
@@ -73,24 +73,32 @@ function SimpleAnimation(
   WAAPIAnimation.propTypes = WAAPIAnimationProps;
 
   const AMPTarget = function ({ children, style = {} }) {
-    const initialStylesFromKeyframes = targetLeafElement
+    // TODO: Support opacity as well?
```

Comment:
I'm not sure if opacity is needed. We initially put this on here to prevent a jump that would occasionally happen in the preview where even tho the animation fill was set to `both` the new page would initially appear without any keyframe styles applied, then a frame later actually apply the initial keyframe styles. This was a brute force approach to prevent that.

If we want to test out if support for opacity is needed here, we could:
-> set `prefers-reduced-motion` disabled
-> create a story with two pages
-> on the second page, add a fg element and give it a fade-in effect
-> preview the story
-> click to the next page
-> see if there's an initial frame where the element is completely visible before playing the fade-in animation

I'm not noticing it being an issue right now with your branch on my local

Reply:
> see if there's an initial frame where the element is completely visible before playing the fade-in animation

There is 😞  You can see the second page here:

![Screenshot 2021-05-19 at 19 40 58](https://user-images.githubusercontent.com/841956/118859275-5a29c400-b8da-11eb-857b-959a5a18414c.png)

Whereas on `main` it's not visible because of the opacity

![Screenshot 2021-05-19 at 19 42 54](https://user-images.githubusercontent.com/841956/118859355-6ada3a00-b8da-11eb-993c-9ddb53a65a0e.png)

(both screenshots taken _without_ Reduce Motion)

Reply:
So in the "preview" of the second page the element is already visible, but there's actually an animation. Here's a screencast of what I mean:

https://user-images.githubusercontent.com/841956/118859718-d7553900-b8da-11eb-915a-9aa4a5fdb057.mov

So you basically see the element's end state in the preview now (due to the lack of opacity)

Reply:
Updated code to include opacity now as well.